### PR TITLE
contributing: Add docs for how to add a benchmark to the PR bench runner

### PR DESCRIPTION
**Pull request by ajgajg1134**
### What does this PR do?
Add details for HOW to add a new benchmark to the PR bench runner. There are further details "to be written" in internal documentation.

### Describe how to test/QA your changes
no QA

### Reviewer's Checklist

- [ ] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] Changed code has unit tests for its functionality.
- [ ] If this interacts with the agent in a new way, a system test has been added.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -61,4 +61,8 @@ The maintainers of this repository assign milestones to pull requests to classif
 
 ### Benchmarks
 
-Some benchmarks will run on any new PR commits, the results will be commented into the PR on completion. To see the list of benchmarks or to add additional benchmarks run go to `.gitlab/scripts/run-benchmarks.sh`.
\ No newline at end of file
+Some benchmarks will run on any new PR commits, the results will be commented into the PR on completion.
+
+#### Adding a new benchmark
+To add additional benchmarks that should run for every PR go to `.gitlab/scripts/run-benchmarks.sh`.
+Add the name of your benchmark to the list in `-bench "BenchmarkConcurrentTracing|BenchmarkStartSpan"` using pipe character separators. Note that your new benchmark must already exist in the `main` branch, for that reason it is best for new benchmarks to be added in their own PR and a second PR opened afterwards to add them to the PR benchmark script.
```
Comment (nsrip-dd) [inline CONTRIBUTING.md:68]: Seems simple enough. Slightly out of scope for this PR, but would it make sense to do `-bench=.` and run every benchmark? Then nothing needs to change when a new benchmark is added.

**Review by nsrip-dd — APPROVED** [review]
Thanks Andrew! One small question, but it's a little out of scope and could be discussed more at the next guild meeting or something if the answer is complicated.

File: CONTRIBUTING.md (hunk 1 above)
Comment (ajgajg1134) [reply CONTRIBUTING.md:68]: We chose to hand-select benchmarks as there's a significant cost in compute to run these benchmarks (and from the look over that Kyle and I did it seems there are a significant number of outdated / not-very-useful benchmarks).

Comment (nsrip-dd) [reply CONTRIBUTING.md:68]: Got it, makes sense to me. Thanks!

**Pull request merged**